desktop/com.github.xournalpp.xournalpp.appdata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2019 Tobias Mueller <tobiasmue@gnome.org>  -->
 <component type="desktop">
-  <id>com.github.xournalpp.xournalpp</id>
+  <id>com.github.xournalpp.xournalpp.desktop</id>
```

Comment:
this is wrong, cf. https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic
the desktop suffix is not part of the appid.

Reply:
Looking at working files from Gnome applications this is widely used though.

Reply:
The `.desktop` suffix seems to be legacy ([Section 2.2.2 of the spec](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html)). So actually this change is unneeded.

Reply:
> Looking at working files from Gnome applications this is widely used though.

what files are you looking at?